android/app/src/main/java/app/organicmaps/sound/TtsPlayer.java

```diff
@@ -164,7 +164,14 @@ public void onDone(String utteranceId) {
         }
 
         @Override
-        public void onError(String utteranceId) {
+        @SuppressWarnings("deprecated") // abstract method must be implemented
```

Comment:
What does it mean? Can the abstract method be implemented instead of suppressing it?

Reply:
The abstract method in the system library was declared as deprecated. You don't have an option not to implement it. This means that the deprecation warning is not possible to avoid.